services/state/file/file.proto

```diff
@@ -29,39 +38,77 @@ option java_package = "com.hederahashgraph.api.proto.java";
 option java_multiple_files = true;
 
 /**
- * Representation of a Hedera Token Service file in the network Merkle tree.
+ * Representation of an Hedera File Service `file`.
+ *
+ * Files offer a place to store additional data, much more than is available in
+ * other entities, for use with smart contracts, non-fungible tokens, etc...
+ * As with all network entities, a file has a unique entity number, which is
+ * given along with the network's shard and realm in the form of a
+ * shard.realm.number id.
  *
- * As with all network entities, a file has a unique entity number, which is given along
- * with the network's shard and realm in the form of a shard.realm.number id.
+ * > Review Question
+ * >> File lacks auto renewal (account and period).  We need to add that here,
+ * >> and also to create and update transactions.
+ * >
+ * >> The keys field is weird and really needs to change to adminKey and
+ * >> deleteKey to be more correct and match other things.
  */
 message File {
     /**
-     * The file's unique file identifier in the Merkle state.
+     * This file's ID within the global network state.
+     * <p>
+     * This value SHALL be unique within the network.
      */
     FileID file_id = 1;
+
     /**
-      * The file's consensus expiration time in seconds since the epoch.
-      */
+     * The file's expiration time in seconds since the epoch.
+     * <p>
+     * For this purpose, `epoch` SHALL be the UNIX epoch with 0 at `1970-01-01T00:00:00.000Z`.
+     */
     int64 expiration_second = 2;
+
     /**
-     * All keys at the top level of a key list must sign to create, modify and delete the file.
+     * A list of keys that MUST sign any transaction to update this file.
+     * <p>
+     * Only _one_ of these keys must sign a transaction to delete the file.<br/>
+     * This field SHOULD NOT be `null` or an empty list.
      */
     KeyList keys = 3;
+
     /**
-     * The bytes that are the contents of the file
+     * The contents of the file.
+     * <p>
+     * This SHALL be limited to the current maximum file size; typically no
+     * more than 1 Megabyte (1048576 bytes).
      */
     bytes contents = 4;
+
     /**
-     * The memo associated with the file (UTF-8 encoding max 100 bytes)
+     * A short description of the file.
+     * <p>
+     * This value, if set, MUST NOT exceed 100 bytes when encoded as UTF-8.
```

Comment:
Should we call out configuration properties when applicable? This value must no exceed `transaction.maxMemoUtf8Bytes` which has a default value of 100 bytes.